dts/arm/nordic/nrf5340_cpuappns.dtsi

```diff
@@ -29,7 +29,7 @@
 	};
 
 	chosen {
-		zephyr,entropy = &rng_hci;
+		zephyr,entropy = &psa_rng;
```

Comment:
Commit missing tag, is this noup, fromlist or fromtree?

Reply:
Will update.